File: Stephannie/Exercise-11.py
print("WELCOME to Pizza Avenue.")

print("We have; \n, 1. Small(S), 2.Medium(M), 3.Large(L)")
pizza_size_choice = int(input("Select from the available options the size you want: "))
if pizza_size_choice not in [1, 2, 3]:
    print("Wrong entry. Make a new selection.")


print("Toppings available are;\n 1.Pepperoni, 2.Magarita, 3.Vegan, 4.Beef and Chicken")
pizza_flavor = int(input("Select your preferred toppings: "))

if  pizza_flavor not in [1, 2, 3, 4]:
    print("Wrong entry. Make a new selection.") 


cheese_option = input("Do you want extra cheese? Y or N.").upper()
if cheese_option not in ["Y", "N"]:
    cheese_option = 0;

Pizza_C = {1: 20, 2 : 30, 3 : 50}

Toppings_C = {1 : 20, 2 : 10, 3 : 15, 4 : 20}

Cheese_C = {"Y": 10, "N": 0}

total_price = Pizza_C[pizza_size_choice] + Toppings_C[pizza_flavor] + Cheese_C[cheese_option]
print(f"The total price of your order is {total_price}")




 

File: Pokua/BasicChallenges/Exercise11.java
package BasicChallenges;

import java.util.Scanner;

public class Exercise11{

    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        System.out.println("Welcome to Papa's Pizza!");

        System.out.println("What size do you want? S, M or L : ");
        String size = input.nextLine();

        System.out.println("Do you want pepperoni? Y or N : ");
        String pepperoni = input.nextLine();

        System.out.println("Do you want extra cheese? Y or N : ");
        String extra = input.nextLine();

        int totalPrice=0;

        if (size.equals("S")) {
            totalPrice += 15;
        } else if (size.equals("M")) {
            totalPrice += 20;
        } else if (size.equals("L")) {
            totalPrice += 25;
        } else {
            System.out.println("Invalid size input.");
        }

        if (pepperoni.equals("Y")) {
            if (size.equals("S")) {
                totalPrice += 2;
            } else {
                totalPrice += 3;
            }
        }

        if (extra.equals("Y")) {
            totalPrice += 1;
        }

        System.out.println("Your final bill is: $" + totalPrice);

        input.close();
    }

}
